### ISSUE TITLE
Browsing the references to a slot do not show references in slot declaration

### ISSUE THREAD

**Issue by jecisc** [guillep/pharo-core#13101]
I often want to check how others are using a slot, but it's hard to find when you cannot search slot declaration :(

**Comment by MarcusDenker** [comment]
Yes, we need to add some kind of UI support... For now we have to #slotUsers:

```
ObservableSlot slotUsers 
```

Another aspect is that full text search should include more than just methods.
